### PR TITLE
Add `random` module to `riot-headers`

### PR DESCRIPTION
**Pull request by Remmirad**
Needed for https://github.com/RIOT-OS/rust-riot-wrappers/pull/54

### CODE REVIEW

File: riot-headers.h [hunk 1]
```diff
@@ -165,11 +168,11 @@
...
-#  include "host/ble_gatt.h"
-#  ifndef IS_C2RUST
-#    include "host/ble_gap.h"
-#  endif
-#  include "host/ble_hs_adv.h"
+#include "host/ble_gatt.h"
+#ifndef IS_C2RUST
+#include "host/ble_gap.h"
+#endif
+#include "host/ble_hs_adv.h"
```
Comment (chrysn) [inline riot-headers.h:175]: Please don't alter unrelated code needlessly.

Comment (Remmirad) [reply riot-headers.h]: fixed, sry did not realize that the auto formatter had kicked in.
